cmd/root.go

```diff
@@ -30,8 +30,10 @@ var (
 
 // RootCmd is the root of cobra subcommand tree
 var RootCmd = &cobra.Command{
-	Use:   "kubeapps",
-	Short: "Kubeapps Installer manages to install Kubeapps components to your cluster",
+	Use:           "kubeapps",
+	Short:         "Kubeapps Installer manages to install Kubeapps components to your cluster",
+	SilenceErrors: true,
```

Comment:
why silencing these?

Reply:
It's just a cobra's tactic for silencing errors down stream. Otherwise the error message will appear twice like this:

```
Error: the server could not find the requested resource
ERRO[0000] the server could not find the requested resource 
```